api/controllers/game/join-rematch.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Join the two players into the new game
+ */
+module.exports = async function (req, res) {
+  try {
+    const { usr: userId } = req.session;
+    const { oldGameId } = req.body;
+
+    const user = await User.findOne({ id: userId });
+
+    const oldGame = await Game.findOne({ id: oldGameId });
```

Comment:
Now that these are independent, we can request them in parallel
```suggestion
    const [ user, game ] = await Promise.all([
      User.findOne({ id: userId }),
      Game.findOne({ id: oldGameId }),
    ]);
```